core/src/main/java/org/mapfish/print/servlet/BaseMapServlet.java

```diff
@@ -117,4 +118,28 @@ protected final StringBuilder getBaseUrl(final HttpServletRequest httpServletReq
         }
         return baseURL;
     }
+
+    /**
+     * Set the cache duration for the queries that can be cached.
+     * @param cacheDurationInSeconds the duration
+     */
+    public final void setCacheDuration(final int cacheDurationInSeconds) {
+        this.cacheDurationInSeconds = cacheDurationInSeconds;
+    }
+
+    /**
+     * Disable caching of the response.
+     * @param response the response
+     */
+    protected void setNoCache(final HttpServletResponse response) {
+        response.setHeader("Cache-Control", "no-cache");
```

Comment:
When I ask Pyramid (python framework we use)  to not allow the caching of its answer it adds that:
```
Cache-Control: max-age=0, must-revalidate, no-cache, no-store
```

And the RFC 7234 says about `private`:
```
A private cache MAY store the response and reuse it for later requests, even if the response
would normally be non-cacheable.
```

We don't want that for our polling URLs, at least.

But OK, Ill change the PR to go halfway in your direction and use the same header as Pyramid.